sdk/core/azure-json/src/main/java/com/azure/json/JsonNull.java

```diff
@@ -0,0 +1,88 @@
+package com.azure.json;
+
+import java.io.IOException;
+
+/**
+ * Class representing the JSON null type
+ */
+public final class JsonNull extends JsonElement {
+
+    // Private constructor enforcing Singleton pattern.
+    private JsonNull() { }
+
+    /**
+     * Helper class to hold Singleton instance.
+     * - Thread-safe lazy-initialization of the JsonNull object without explicit
+     * synchronization.
+     */
+    private static class LoadSingleton {
```

Comment:
Same comment as `JsonBoolean`, the object is cheap to create so we don't need to defer instantiation.

Reply:
This should now be done @alzimmermsft